rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -226,7 +227,7 @@ public Integer getValue()
       @Override
       public Long getValue()
```

Comment:
should we add 'KB' in metric as well?

Reply:
CACHE_AVAILABLE_SIZE_GAUGE needs to be updated from `rubix.bookkeeper.gauge.available_cache_size_mb` to `rubix.bookkeeper.gauge.available_cache_size_kb`

Reply:
I think better to revert this and keep user visible things in MB. Easier to look at values in MB than in KB, will change this part.